src/databaseTest/java/net/sf/jabref/shared/DBMSConnectorTest.java

```diff
@@ -26,12 +27,9 @@
     public void testGetNewConnection() throws ClassNotFoundException, SQLException {
         DBMSConnectionProperties properties = TestConnector.getConnectionProperties(dbmsType);
 
-        try {
+        AssertUtil.assertDoesNotThrow(() -> {
```

Comment:
I don't think that this is necessary. Either an exception is provoked by a test - or it is "something unexpected".
Thus: Annotate the expected excpetion for tests which should throw exceptions and do not catch exceptions in the tests...